README.md

```diff
@@ -9,43 +9,42 @@ This toolkit allows developers to deploy pre-trained deep learning models
 through a high-level C++ Inference Engine API integrated with application logic.
 
 This open source version includes several components: namely [Model Optimizer], [nGraph] and
-[Inference Engine], as well as CPU, GPU, MYRIAD, multi device and heterogeneous plugins to accelerate deep learning inferencing on Intel® CPUs and Intel® Processor Graphics.
-It supports pre-trained models from the [Open Model Zoo], along with 100+ open
-source and public models in popular formats such as Caffe\*, TensorFlow\*,
-MXNet\* and ONNX\*.
+[Inference Engine], as well as CPU, GPU, MYRIAD, multi device and heterogeneous plugins to accelerate deep learning inferencing on Intel® CPUs and Intel® Processor Graphics. It supports pre-trained models from the [Open Model Zoo], along with 250+ open
+source and public models in popular formats such as Caffe, TensorFlow, PyTorch,
+MXNet and ONNX.
 
 ## Repository components:
 * [Inference Engine]
 * [nGraph]
 * [Model Optimizer]
+* [Post-training Optimization Tool]
 
 ## License
-Deep Learning Deployment Toolkit is licensed under [Apache License Version 2.0](LICENSE).
+OpenVINO™ toolkit is licensed under [Apache License Version 2.0](LICENSE).
 By contributing to the project, you agree to the license and copyright terms therein
 and release your contribution under these terms.
 
 ## Resources:
-* Docs: https://docs.openvinotoolkit.org/
+* Docs: https://docs.openvino.ai
 * Wiki: https://github.com/openvinotoolkit/openvino/wiki
 * Issue tracking: https://github.com/openvinotoolkit/openvino/issues
 * Storage: https://storage.openvinotoolkit.org/
 * Additional OpenVINO™ modules: https://github.com/openvinotoolkit/openvino_contrib
-* [Intel® Distribution of OpenVINO™ toolkit Product Page](https://software.intel.com/content/www/us/en/develop/tools/openvino-toolkit.html)
+* [Intel® Distribution of OpenVINO™ toolkit Product Page](https://software.intel.com/openvino)
 * [Intel® Distribution of OpenVINO™ toolkit Release Notes](https://software.intel.com/en-us/articles/OpenVINO-RelNotes)
 
 ## Support
 Please report questions, issues and suggestions using:
 
-* The [`openvino`](https://stackoverflow.com/questions/tagged/openvino) tag on StackOverflow\*
-* [GitHub* Issues](https://github.com/openvinotoolkit/openvino/issues)
+* The [`openvino`](https://stackoverflow.com/questions/tagged/openvino) tag on StackOverflow
+* [GitHub Issues](https://github.com/openvinotoolkit/openvino/issues)
 * [Forum](https://software.intel.com/en-us/forums/computer-vision)
```

Comment:
This forum is empty, Should we change it to https://community.intel.com/t5/Intel-Distribution-of-OpenVINO/bd-p/distribution-openvino-toolkit ?